config/formats.js

```diff
@@ -89,7 +89,7 @@ exports.Formats = [
 		banlist: ['RU', 'BL3', 'Drought'],
 	},
 	{
-		name: "[Gen 7] PU (alpha)",
+		name: "[Gen 7] PU (beta)",
 
 		mod: 'gen7',
 		ruleset: ['[Gen 7] NU (beta)'],
```

Comment:
Yes, this line; you didn't change it. It's not just a title because the higher tier is used as the next tier's ruleset.
Anyway, looks like Kris already handled this and more in https://github.com/Zarel/Pokemon-Showdown/pull/3864.